concepts/commerce/checkout-concept/payments.md

```diff
@@ -80,6 +80,14 @@ This step is only executed for asynchronous payments. It is triggered by the cal
 The actual implementation of payment integrations differs between providers. Therefore, our specification does not include any guidelines about payment states or specific API calls to be made. Some integrations share data between the steps or provide and call upon webhooks after the payment process has been finished. These implementations go beyond our standards.
 :::
 
+### Session and state
```

Comment:
@shyim I added these two small sections. Please take a look.

As for the SalesChannelContextPersister - there is nothing yet about it in the docs. However, there is a store-api route for that:

 #[Route(path: '/store-api/context', name: 'store-api.switch-context', methods: ['PATCH'])]
 
 Should we add that somewhere?